README.md

```diff
@@ -30,38 +6,22 @@ After you're finished please remove all the comments and instructions!
   <h3><b>Microverse README Template</b></h3>
 
 </div>
-
-<!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Hello-rails-with-react] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+> This is a project about creating two linked apps with a rails back-end and react-front-end.
 
-**[your_project__name]** is a...
+# Front-End
+[Link to front-end](https://github.com/Elisha-Tetteyfio/hello-react-front-end)
 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you also provide a link to the Rails API (backend app) in the README file for the front end. This was supposed to be mentioned as a required change by the previous reviewer.

- [ **OPTIONAL** ] Kindly make sure to use **[Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)** for your next projects where it is required to be used. You should follow **[the project's requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)** and apply everything mentioned.